<ui version="4.0" >
 <class>addrimpexp</class>
 <widget class="QDialog" name="addrimpexp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Address import and export...</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/images/first4.qrc" >:/impexp.png</iconset>
  </property>
  <widget class="QStackedWidget" name="mainstack" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>811</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="page" >
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Please select a task:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>791</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title" >
      <string/>
     </property>
     <widget class="QRadioButton" name="rbtn2" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>481</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string>Export vCard</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn3" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>481</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string>Import CSV</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn4" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>100</y>
        <width>481</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string>Export CSV</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn1" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string>Import vCard</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2" >
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Import vCards:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Seite" >
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Export vCard:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Seite_2" >
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Import CSV:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Seite_3" >
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Export CSV:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Directory:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbexpcsv" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>40</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnexpcsvload" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>40</y>
       <width>31</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/images/first4.qrc" >:/button_ok.png</iconset>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>791</width>
       <height>471</height>
      </rect>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
    <widget class="QRadioButton" name="rbtnexpcsvall" >
     <property name="geometry" >
      <rect>
       <x>580</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnexpcsvsel" >
     <property name="geometry" >
      <rect>
       <x>680</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Selected</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnnext" >
   <property name="geometry" >
    <rect>
     <x>724</x>
     <y>610</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Next >></string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback" >
   <property name="geometry" >
    <rect>
     <x>610</x>
     <y>610</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;&lt; &amp;Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>610</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Cancel</string>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/button_cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images/first4.qrc" />
 </resources>
 <connections/>
</ui>
